client/lib/analytics/track-component-view/index.jsx

```diff
@@ -22,8 +22,7 @@ class TrackComponentView extends Component {
 		bumpStat: () => {},
 	};
 
-	// @TODO: Please update https://github.com/Automattic/wp-calypso/issues/58453 if you are refactoring away from UNSAFE_* lifecycle methods!
-	UNSAFE_componentWillMount() {
+	componentDidMount() {
 		debug( 'Component will mount.' );
```

Comment:
Nit: change this debug statement to `did` as well or just remove it.

Reply:
Of course!

I'd remove them all from Calypso if you ask me 😅  but someone must find them useful still, so I've just updated it for now.

Thanks!